app/main/views/marketplace.py

```diff
@@ -172,19 +173,15 @@ def list_opportunities(framework_slug):
     api_result = form.get_briefs()
 
     briefs = api_result["briefs"]
-    links = api_result["links"]
+    meta = api_result['meta']
 
-    api_prev_link_args = parse_link(links, "prev")
-    prev_link_args = None
-    if api_prev_link_args:
-        prev_link_args = request.args.copy()
-        prev_link_args.setlist("page", api_prev_link_args.get("page") or ())
+    results_per_page = meta.get('per_page')
+    total_results = meta.get('total')
+    current_page = int(request.args.get('page', 1))
 
-    api_next_link_args = parse_link(links, "next")
-    next_link_args = None
-    if api_next_link_args:
-        next_link_args = request.args.copy()
-        next_link_args.setlist("page", api_next_link_args.get("page") or ())
+    pages = [1]
```

Comment:
I think the code reads more like what you mean if you write it like

if foo:
 pages = bar
else:
 pages = [1]

rather than setting pages when you're going to override the value in most cases.  Not a big deal, but it makes it easier for other people reading the code.